torch/_dynamo/variables/misc.py

```diff
@@ -456,6 +456,11 @@ def call_method(
             return self.call_apply(tx, args, kwargs)
         elif name == "backward":
             with tx.strict_translation_mode():
+                # backward function can be decorated by torch.autograd.function.once_differentiable,
+                # however, torch.compile doesn't support higher order gradients. And we can assume that
+                # the grad_mode during the backwards pass is False, so just inline the original function.
+                if hasattr(self.fn_cls.backward, "__wrapped__"):
+                    self.fn_cls.backward = self.fn_cls.backward.__wrapped__
```

Comment:
once_differentiable might add `__wrapped__`, but it's not the only way to get `__wrapped__`. functools.wraps adds the `__wrapped__` attribute.

Is there some way to target this change to once_differentiable?

Reply:
I added another check on top of this, to check if this function is wrapped by the function defined in ```torch.autograd.function```. Right now only ```once_differentiable``` is the case. Does it looks good to you?

Reply:
If users wrap ```backward``` function in their own file, the above check won't pass.